plugins/amazonq/codewhisperer/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererFeatureConfigServiceTest.kt

```diff
@@ -80,7 +98,7 @@ class CodeWhispererFeatureConfigServiceTest {
             on { listFeatureEvaluations(any<ListFeatureEvaluationsRequest>()) } doReturn ListFeatureEvaluationsResponse.builder().featureEvaluations(
                 listOf(
                     FeatureEvaluation.builder()
-                        .feature(CodeWhispererFeatureConfigService.CUSTOMIZATION_ARN_OVERRIDE_NAME)
+                        .feature("customizationArnOverride")
```

Comment:
revert?

Reply:
All of the other feature consts were made private so moved this to private which required the hardcoded value

Reply:
i feel it's not necessary to make the experiemtn names private tho

Reply:
it's easier to restructure and refactor featureConfigService to simply use enum as experiment, but it's not blocker for the PR